src/app/EventManagement.cpp

```diff
@@ -33,6 +33,26 @@ namespace chip {
 namespace app {
 static EventManagement sInstance;
 
+#if CHIP_SYSTEM_CONFIG_NO_LOCKING
+
+class EventLocked
+{
+public:
+    EventLocked() {}
+    ~EventLocked() {}
+};
+
+#else
+System::Mutex sEventLock;
```

Comment:
Why is this not a class static in the EventLocked class, instead of being a global?

Reply:
add static, done, thanks